src/query/functions-v2/src/scalars/string.rs

```diff
@@ -311,6 +342,144 @@ pub fn register(registry: &mut FunctionRegistry) {
             },
         ),
     );
+
+    // number to string functions
+    let all_numerics_types = &[
+        DataType::UInt8,
+        DataType::UInt16,
+        DataType::UInt32,
+        DataType::UInt64,
+        DataType::Int8,
+        DataType::Int16,
+        DataType::Int32,
+        DataType::Int64,
+        DataType::Float32,
+        DataType::Float64,
+    ];
+
+    for dtype in all_numerics_types {
+        with_unsigned_number_mapped_type!(T, match dtype {
+            DataType::T => {
+                registry.register_1_arg::<NumberType<T>, StringType, _, _>(
+                    "bin",
+                    FunctionProperty::default(),
+                    |_| None,
+                    |val| format!("{:b}", val as u64).as_bytes().to_vec(),
```

Comment:
maybe we just need to implement `u64` version. 

For other unsigned numbers, the evaluator will cast them into u64 in `type_checker`.

Reply:
How about signed integer type and float type?

Reply:
signed integer types  --> i64
float types -> f64

We have a special case for  `arithmetics` because they are commonly used.

For other numeric arguments, it's ok just to declare the largest type.

Reply:
Understood.